mlflow/store/tracking/rest_store.py

```diff
@@ -403,16 +404,35 @@ def _delete_traces(
         max_traces: int | None = None,
         trace_ids: list[str] | None = None,
     ) -> int:
-        req_body = message_to_json(
-            DeleteTraces(
-                experiment_id=experiment_id,
-                max_timestamp_millis=max_timestamp_millis,
-                max_traces=max_traces,
-                request_ids=trace_ids,
+        # If deleting by trace_ids and the number exceeds the batch size limit,
+        # split into batches to avoid hitting the Databricks server limit
+        if trace_ids and len(trace_ids) > _MLFLOW_DELETE_TRACES_MAX_BATCH_SIZE.get():
```

Comment:
```suggestion
        if trace_ids and len(trace_ids):
```

we can always batch when trace_ids is truthy.

Reply:
Done